maintainers/scripts/update-luarocks-packages

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -p nix-prefetch-scripts luarocks-nix -i bash
+#!nix-shell -p parallel nix-prefetch-scripts luarocks-nix -i bash
```

Comment:
Didn't you want to add a shell-luarocks.nix to force using the one from the same nixpkgs ?

Reply:
Yeah, I forgot :p. I'll add a commit for it.